README.md

```diff
@@ -1,168 +1,222 @@
 # DSP-UI library
 
-This is the demo and developing environment for DSP-UI library (@knora/ui) composed of 4 modules.
+This is the demo and developing environment for DSP-UI-LIB library (@dasch-swiss/dsp-ui) composed of 4 modules.
 
-The modules helps to create a graphical user interface, a web application to use [DSP-API](https://www.knora.org) in a quick and simple way. The modules are written in typescript to use them with [Angular](https://angular.io) (version 8). We decided to style the components and directives with [Angular Material design](https://material.angular.io).
+The modules help to create a graphical user interface, a web application to use [DSP-API](https://www.knora.org) in a quick and simple way. The modules are written in Typescript to use them with [Angular](https://angular.io) (version 9). We decided to style components and directives with [Angular Material design](https://material.angular.io).
 
-DSP-UI library implements [DSP-API-JS-LIB](https://www.npmjs.com/package/@knora/api) to connect with the Knora web API. DSP-API is a software framework for storing, sharing, and working with primary sources and data in the humanities.
+DSP-UI-LIB implements [DSP-JS-LIB](https://www.npmjs.com/package/@knora/api) to connect with the DSP API. DSP-API is a software framework for storing, sharing, and working with primary sources and data in the humanities.
 
-DSP-API and the DSP-UI library is [free software](http://www.gnu.org/philosophy/free-sw.en.html), released under the [GNU Affero General Public](http://www.gnu.org/licenses/agpl-3.0.en.html).
+DSP-API and DSP-UI-LIB are [free software](http://www.gnu.org/philosophy/free-sw.en.html), released under the [GNU Affero General Public](http://www.gnu.org/licenses/agpl-3.0.en.html).
 
-This version of DSP-UI requires [DSP-API version ^12.0.0](https://github.com/dasch-swiss/knora-api/releases/tag/v12.0.0).
+This version of DSP-UI-LIB requires [DSP-API version ^12.0.0](https://github.com/dasch-swiss/knora-api/releases/tag/v12.0.0).
 
 ## Library modules
 
-### DSP-UI/core
+### DspCoreModule
 
 The core module contains configuration files and all variables needed to connect the DSP API.
 [read more...](https://dasch-swiss.github.io/knora-ui/modules/core)
```

Comment:
What is the source of https://dasch-swiss.github.io/knora-ui/modules/core?

Reply:
Does this refer to the "old" knora-ui?

Reply:
At the moment: yes
I also thought to hide stuff, which is not yet implemented in the upcoming release. 
@flavens We should comment them out?

Reply:
I strongly suggest to keep this separate.

Reply:
just mention these things as future steps (roadmap)

Reply:
Yes, we have it in mind...